pkg/apis/authentication/v1alpha1/types.go

```diff
@@ -9,27 +9,27 @@ import (
 // +genclient
```

Comment:
You need some magic to tell it the scope of the resource.

```
// +genclient
// +genclient:nonNamespaced
// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

Reply:
Thanks, I was wondering something would probably have to be changed here in order to get this non-namespaced as I went through the generated code, but I have literally no knowledge about these directives.